package.json

```diff
@@ -1,11 +1,15 @@
 {
   "name": "osmintegrator-frontend",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.4.1",
+    "@emotion/styled": "^11.3.0",
     "@material-ui/core": "^4.12.3",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.60",
+    "@mui/icons-material": "^5.0.0",
+    "@mui/material": "^5.0.0",
```

Comment:
Do we really want to keep 2 different version of material (4.x vs 5.x) ?

Reply:
It's just for icons ;)